node/src/main/java/eu/cloudnetservice/node/command/sub/ServiceCommand.java

```diff
@@ -419,9 +421,9 @@ private void displayServiceInfo(
 
   private @NonNull Collection<String> parseExcludes(@Nullable String excludes) {
     if (excludes == null) {
-      return Collections.emptyList();
+      return List.of();
     }
 
-    return Arrays.asList(excludes.split(";"));
+    return List.of(excludes.split(";"));
```

Comment:
I think we should replace that parsing logic with a guava splitter which can omit some common mistakes already:
`private static final Splitter SEMICOLON_SPLITTER = Splitter.on(';').omitEmptyStrings().trimResults();`

and this method like:
`return excludes == null ? List.of() : SEMICOLON_SPLITTER.splitToList(excludes);`

And as this method is now used multiple times for not only parsing excludes we should rename it and the parameter.